README.md

```diff
@@ -1,24 +1,212 @@
+
+- Have ruby installed
+- Have node installed
+- Have rails installed
+
+### Setup
+
+You can find the React Frontend client at [React_client](https://github.com/j-tee/front-end-booking-app/tree/development)
+
+Clone this repository to your desired folder:
+
+To run and view this application, you will first have to clone a local version of this repository using:
+
+`git clone https://github.com/j-tee/booking-app.git`
+
+After that navigate to the newly created folder:
+
+`cd booking-app`
+
+### Install
+
+Install this project with:
+ `bundle install`
+
+### Usage
+
+To run the project, execute the following command:
+
+You can find the React Frontend client at [React_client](https://github.com/RayhanTabase/hello-react-front-end)
+
+Configure PostgreSQL database in your project in the project directory:
+
+`config/database.yml`
+
+Start a local server running the command:
+
+`rails s`
+
+
+### Run tests
+
+To run tests, run the following command:
+
+`bundle exec rspec`
+
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Salim Abdulai**
+
+- GitHub: [@RayhanTabase](https://github.com/RayhanTabase)
+- LinkedIn: [Salim-Abdulai](https://linkedin.com/in/salim-abdulai-5430065b)
+
+👤 **Samuel**
+
+- GitHub: [@devMunyi](https://github.com/devMunyi)
+- LinkedIn: [Samuel Munyi](https://www.linkedin.com/in/samuel-munyi)
+
+👤 **Julius**
+
+- GitHub: []()
+- LinkedIn: []()
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[authentication]**
+- [ ] **[authorizations]**
+- [ ] **[sessions]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+I would like to thank:
+- Microverse for providing an excellent curriculum that has helped me enhance my skills in software development.
+- Code Reviewers for their valuable feedback and guidance throughout my learning journey.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪